### PR TITLE
[W03H03] Add corner case test in build_seams_Seamsarray

### PR DESCRIPTION
**Pull request by David-Petrov**
feat(w03h03/test/pgdp/SeamCarvingTest.java): 

Hey, ich hab kurz an folgenden Corner Case gedacht: 
welcher Pixel ist aus der nächsten Zeile zu bevorzugen, wenn man sich gegenwärtig am ganz rechten Rand einer Zeile befindet und in der unteren Zeile die beiden rechtesten Zahlen gleich sind?

Z.B. hier, von der 10 in der ersten Zeile: soll die linke oder die rechte 5 gewählt werden?

| Rest von links  | Vorletzt | Letzt |
| ------------- | ------------- | - |
| ... | _ | 10 |
| ... | 5 | 5 |

Was hier passieren soll, wird mir aus der Aufgabe aber nicht ganz klar. Da steht ja, dass bei gleichen Zahlen "links" und "rechts" immer die linke gewählt werden soll, dagegen aber auch immer "die mittlere", wenn sie unter den niedrigsten ist... welche unter zwei Zahlen die mittlere ist, ist nicht ganz klar 😄 

Ich würde mal sagen, der Seam strebt immer optimal "nur gerade nach unten" zu verlaufen, daher hab ich auch den test so geschrieben. Alles andere passt mit dieser Implementation. Was denkt ihr?

### CODE REVIEW

**Comment by felixhauptmann** [comment]
Sollte definitiv die 5 in der Mitte, also direkt darunter sein

File: w03h03/test/pgdp/SeamCarvingTest.java [hunk 1]
```diff
@@ -158,20 +158,43 @@ void testing_build_seams_seamsArray(int[][] seams, long[] seamWeights, int[] gra
...
-                            },
-                            6,
-                            5,
-                            new int[][]{new int[]{0, 0, 0, 1, 1}, new int[]{1, 1, 2, 1, 1}, new int[]{2, 2, 2, 1, 1}, new int[]{3, 3, 3, 4, 4}, new int[]{4, 4, 4, 4, 4}, new int[]{5, 5, 4, 4, 4}}
-                    )
+                arguments(
+                    new int[6][5],
+                    new long[6],
+                    new int[] {
+                        Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE, Integer.MAX_VALUE,
```
Comment (felixhauptmann) [inline w03h03/test/pgdp/SeamCarvingTest.java:165]: please reformat so every line has width elements (it was properly formatted before)

**Review by felixhauptmann — CHANGES_REQUESTED** [review]

File: w03h03/test/pgdp/SeamCarvingTest.java (hunk 1 above)
Comment (David-Petrov) [reply w03h03/test/pgdp/SeamCarvingTest.java:165]: > please reformat so every line has width elements (it was properly formatted before)

I don't exactly get what you mean by 'width elements'. Indentation or the array initialization? Anyways, feel free to format it as you like. 😄
